bin/reth/src/main.rs

```diff
@@ -0,0 +1,52 @@
+#![doc = include_str!("../README.md")]
+#![doc(issue_tracker_base_url = "https://github.com/paradigmxyz/op-rs/issues/")]
+#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
```

Comment:
Can we add the `unused_crate_dependencies` doc lint

```suggestion
#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
#![cfg_attr(not(test), warn(unused_crate_dependencies))]
```